infrastructures/docker/src/main/java/org/eclipse/che/workspace/infrastructure/docker/DockerInternalRuntime.java

```diff
@@ -203,29 +208,39 @@ protected void internalStart(Map<String, String> startOptions) throws Infrastruc
             }
         } catch (InfrastructureException | InterruptedException | RuntimeException e) {
```

Comment:
Haven't we replaced usage of interrupted exception with RuntimeStartInterruptedException in this PR?